### PR TITLE
issue #160: Better command line printout for run bundle builder 

### PR DESCRIPTION
**Pull request by ChengshuLi**
@percyliang 

#160

### CODE REVIEW

File: codalab/apps/web/static/js/worksheet/run_bundle_builder.jsx [hunk 1]
```diff
@@ -85,24 +85,19 @@ var RunBundleBuilder = React.createClass({
...
-      } else {
-        target = target.path === '' ? target.bundle_name : target.bundle_name + '/' + target.path;
-      }
+      target = target.path === '' ? shorten_uuid(target.bundle_uuid) : shorten_uuid(target.bundle_uuid) + '/' + target.path;
       clCommand.push(key + ':' + target);
     }
     if (command != null) {
-      command = command.replace(/"/g, '\\\"');
-      clCommand.push("\"" + command + "\"");
+      command = buildTerminalCommand(command.split(' '))
```
Comment (percyliang) [inline codalab/apps/web/static/js/worksheet/run_bundle_builder.jsx]: missing semicolon

File: codalab/apps/web/static/js/worksheet/run_bundle_builder.jsx [hunk 2]
```diff
@@ -85,24 +85,19 @@ var RunBundleBuilder = React.createClass({
...
+    var clCommand = ['run'];
     for (var i = 0; i < dependencyKeyList.length; i++) {
       var key = dependencyKeyList[i];
       var target = selectedDependencies[i];
-      if (usingBundleId) {
-        target = target.path === '' ? target.bundle_uuid : target.bundle_uuid + '/' + target.path;
-      } else {
-        target = target.path === '' ? target.bundle_name : target.bundle_name + '/' + target.path;
-      }
+      target = target.path === '' ? shorten_uuid(target.bundle_uuid) : shorten_uuid(target.bundle_uuid) + '/' + target.path;
```
Comment (percyliang) [inline codalab/apps/web/static/js/worksheet/run_bundle_builder.jsx]: rewrite so shorten_uuid appears once

File: codalab/apps/web/static/js/worksheet/run_bundle_builder.jsx [hunk 3]
```diff
@@ -6,7 +6,7 @@ var RunBundleBuilder = React.createClass({
       selectedDependencies: [],
       dependencyKeyList: [],
       command: null,
-      clCommand: 'cl run ',
+      clCommand: 'run',
```
Comment (percyliang) [inline codalab/apps/web/static/js/worksheet/run_bundle_builder.jsx]: Why does this need to be here, since `clCommand` is a local variable below...

**Comment by percyliang** [comment]
@ChengshuLi see comments

**Comment by ChengshuLi** [comment]
@percyliang 
fixed

**Comment by percyliang** [comment]
LGTM

**Pull request merged**